docs/en/getting-started/installation.md

```diff
@@ -290,7 +290,7 @@ For the file system that has been mounted using the old version of JuiceFS clien
 
 ## Uninstall
 
-The JuiceFS client only has one binary file, so it can be easily deleted once you find the location of the program. For example, to delete the client that is installed on the Linux system following this document, you only need to execute the following command to uninstall the client.
+The JuiceFS client only has one binary file, so it can be easily deleted once you find the location of the program. For example, to delete the client that is installed on the Linux system as described above, you only need to execute the following command:
```

Comment:
```suggestion
The JuiceFS client has only one binary file, so it can be easily deleted once you find the location of the program. For example, to uninstall the client that is installed on the Linux system as described above, you only need to execute the following command:
```